src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -401,7 +399,71 @@ void consumeRecords(final List<O> recOps) {
     }
   }
 
-  void createTopics(final List<O> topicOps) {}
+  void createTopics(final List<O> topicOps) {
```

Comment:
The method is too long. Please consider the refactoring (see the comments below).